tests/InputFiles/CurvedScalarWave/PlaneWaveMinkowski3D.yaml

```diff
@@ -1,18 +1,21 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
-# Executable: EvolveCurvedScalarWavePlaneWaveMinkowski3D
-# Check: parse;execute_check_output
-# Timeout: 15
-# ExpectedOutput:
-#   PlaneWaveMinkowski3DVolume0.h5
-#   PlaneWaveMinkowski3DReductions.h5
-# OutputFileChecks:
-#   - Label: constraint violations
-#     Subfile: /Norms.dat
-#     FileGlob: PlaneWaveMinkowski3DReductions.h5
-#     SkipColumns: [0, 1, 2]
-#     AbsoluteTolerance: 0.02
+Executable: EvolveCurvedScalarWavePlaneWaveMinkowski3D
+Testing:
+  Check: parse;execute_check_output
+  Timeout: 15
+ExpectedOutput:
+  - PlaneWaveMinkowski3DVolume0.h5
+  - PlaneWaveMinkowski3DReductions.h5
+OutputFileChecks:
+  - Label: constraint violations
+    Subfile: /Norms.dat
+    FileGlob: PlaneWaveMinkowski3DReductions.h5
+    SkipColumns: [0, 1, 2]
+    AbsoluteTolerance: 0.02
```

Comment:
I've been going back and forth on how I feel about having testing stuff at the top of these input files. Currently, we use the "test" input files also as the "example" input files that we point users to. I think it would be nice to keep these input files where they are and have them be used for testing only. These input files would have the `Testing:` block in the metadata like it does here. Then, separately, we have a directory `$SPECTRE_HOME/ExampleInputFiles/` where we hold input files that don't have the `Testing:` block (or `ExpectedOutput:` or `OutputFileChecks`). 

The downside to this is that these input files wouldn't be checked automatically (or we add special CMake magic to only run a parse test on all of them). I'm not sure which one is better because from a user perspective, having the example input files in the root of the repo (I think) makes the most logical sense rather than in the `test` directory. However, it may be more work for us to maintain two copies of some input files.

Another pro for having the `ExampleInputFiles/` directory is we can set some reasonable defaults for all inputs like domain, time stepping, etc. We could even add EventsAndTriggers we'd like people to run (because they already just copy the input files anyways, that way they are copying something we want them to use).

Either way, I think the `ExpectedOutput:` and `OutputFileChecks:` blocks should be nested inside the `Testing:` block because they are only used for testing.

Reply:
Quick reply to this part:

> Either way, I think the ExpectedOutput: and OutputFileChecks: blocks should be nested inside the Testing: block because they are only used for testing.

These two blocks are used by the `CleanOutput` and `CheckOutputFiles` tools, respectively. At least `CleanOutput` is used outside of tests, e.g. to clean up a failed run before rerunning it. My draft of a scheduling script also invokes`CleanOutput` before running if passed a "cleanup" flag. Since these are separate tools, I chose to keep their options outside the `Testing` block. The `Testing` block is what's read by CMake.

Reply:
Ah ok, that seems reasonable. If the `CheckOutputFiles` block is actually only used for testing, I think it should be moved, but I don't know what your plans are for the scheduler.

Reply:
`CheckOutputFiles` is mostly used for testing, but it's an independent tool so you might use it outside of tests to run checks over you output data. E.g., check that the shift is zero to some tolerance in a stable TOV evolution or something like that. As I wrote above, the `Testing` block is what's read by CMake to set up the input file tests.